### PR TITLE
ICU-20979 API Change Report for 67 RC

### PR DESCRIPTION
**Pull request by srl295**
[ICU-20979]

API Change Report… 66 vs 67

[ICU-20979]: https://unicode-org.atlassian.net/browse/ICU-20979

### CODE REVIEW

File: icu4c/APIChangeReport.md [hunk 1]
```diff
@@ -37,55 +43,178 @@ Changed in  ICU 66 (old, new)
...
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_LISU_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_SYMBOLS_FOR_LEGACY_COMPUTING |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_TANGUT_SUPPLEMENT |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UBlockCode::UBLOCK_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
-| uchar.h | <tt>enum</tt> UIndicPositionalCategory::U_INPC_TOP_AND_BOTTOM_AND_LEFT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_CHORASMIAN |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_DIVES_AKURU |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_KHITAN_SMALL_SCRIPT |  (missing) | StableICU 66| *(Born Stable)* |
-| uscript.h | <tt>enum</tt> UScriptCode::USCRIPT_YEZIDI |  (missing) | StableICU 66| *(Born Stable)* |
+| bytestream.h | void icu::ByteSink::AppendU8(const char*, int32_t) |  (missing) | DraftICU 67
```
Comment (sffc) [inline icu4c/APIChangeReport.md]: This table does not render after the first row

File: icu4c/APIChangeReport.md [hunk 2]
```diff
@@ -203,7 +336,19 @@ Other existing drafts in ICU 66
 | locid.h |  icu::Locale::ConvertingIterator&lt; Iter, Conv &gt;::ConvertingIterator(Iter, Iter, Conv) | DraftICU 65 | 
 | locid.h |  icu::Locale::Iterator::~Iterator() | DraftICU 65 | 
 | locid.h |  icu::Locale::RangeIterator&lt; Iter &gt;::RangeIterator(Iter, Iter) | DraftICU 65 | 
+| locid.h |  void icu::Locale::canonicalize(UErrorCode&amp;) | 
 | measfmt.h |  void icu::MeasureFormat::parseObject(const UnicodeString&amp;, Formattable&amp;, ParsePosition&amp;) const | DraftICU 53 | 
+| measunit.h |  MeasureUnit icu::MeasureUnit::product(const MeasureUnit&amp;, UErrorCode&amp;) const |
```
Comment (sffc) [inline icu4c/APIChangeReport.md]: These APIs are added; why are they in the pre-existing section?

File: icu4c/APIChangeReport.md (hunk 1 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: I saw that

File: icu4c/APIChangeReport.md (hunk 2 above)
Comment (pedberg-icu) [reply icu4c/APIChangeReport.md]: Shouldn't this wait until after the API status update?

Comment (srl295) [reply icu4c/APIChangeReport.md]: please recheck

**Comment by sffc** [comment]
How about promotions from draft 64 to stable 67?

File: icu4c/APIChangeReport.md (hunk 1 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: fixed now?

**Comment by srl295** [comment]
> How about promotions from draft 64 to stable 67?

I think that's a separate task?

**Comment by srl295** [comment]
Will update once #1081 is merged

**Comment by markusicu** [comment]
> Will update once #1081 is merged

That's done. Please update.

File: icu4c/APIChangeReport.md (hunk 2 above)
Comment (srl295) [reply icu4c/APIChangeReport.md]: regenerated

**Comment by srl295** [comment]
OK… 

https://github.com/unicode-org/icu/commits/master <https://github.com/unicode-org/icu/commits/master> shows the last commit as https://github.com/unicode-org/icu/commit/94c2c578a92b287772637ad6f7a30c2280a1a661

You can see in the branch that the PR points to, that the commit is included: https://github.com/srl295/icu/commits/icu4c-api-67-rc <https://github.com/srl295/icu/commits/icu4c-api-67-rc>

I did a clean rebuild, the only thing that changed was the date. Investigating…

--
Steven R. Loomis | @srl295 | git.io/srl295

**Comment by markusicu** [comment]
The new BasicTimeZone operator= is protected.
Does the API change report ignore protected API?

**Pull request merged**